src/chrome/create-chrome.ts

```diff
@@ -166,11 +165,13 @@ export const createChromeContext = ({
     on,
     experimentalApi: true,
     isFedramp: isITLessEnv,
-    usePendoFeedback,
+    usePendoFeedback: () => chromeStore.set(usePendoFeedbackAtom, false),
```

Comment:
This should not be false. You have to imitate the logic of the previous `usePendoFeedback` hook.